paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUI.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.unit.dp
+import com.stripe.android.paymentsheet.DisplayableSavedPaymentMethod
+import com.stripe.android.paymentsheet.R
+
+@Composable
+internal fun ManageScreenUI(interactor: ManageScreenInteractor) {
```

Comment:
I think long term we will want both screenshot and non screenshot unit tests for this.

Right now the only non screenshot test that really makes sense to me is something like [this](https://github.com/stripe/stripe-android/blob/79b0b4a0fa32055e233b4d534fc254c9b3587b94/paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUITest.kt#L57).

Reply:
just added a test like that! Thanks